nav-app/src/app/matrix/matrix-flat/matrix-flat.component.html

```diff
@@ -32,9 +32,10 @@
                     <ng-template #supertechnique>
                         <table class="supertechnique">
                             <tr>
-                                <td class="sidebar technique" (click)="onToggleSubtechniquesVisible(technique, tactic)">
+                                <td *ngIf="applyControls(technique.subtechniques, tactic).length > 0" class="sidebar technique" (click)="onToggleSubtechniquesVisible(technique, tactic)">
                                     <div class="handle"> = </div>
                                 </td>
+                                <td *ngIf="technique.subtechniques.length > 0 && applyControls(technique.subtechniques, tactic).length == 0" class="sidebar technique disabled"><div class="handle"> = </div></td>
```

Comment:
IS there a reason you're doing a "disabled" vs "non-disabled" element, and not doing a `[ngClass]="{disabled: technique.subtechniques.length > 0 && ... }" on the same element?